templates/default/note/note.html

```diff
@@ -181,4 +181,18 @@ <H2 align="center" class="hdf">
 </TABLE>
 {/if}
 
+{if !ConfigHelper::checkConfig('notes.issuers_signature')}
```

Comment:
Proponuję zmianę **issuers_signature** na **issuer_signature** - zawsze tak dajemy.
Rozumiem, że wersje PDF not trzeba wyposażyć w nową funkcję?

Reply:
Poprawię na **issuer**. Na początku tak, zrobiłem ale potem zmieniłem.

Właśnie co do PDF. Jak to wygląda? Bo faktury mają osobny moduł, który rysuje PDF, a do innych dokumentów kiedyś dorobiłeś bezgłowego Chrome'a zdaje się, który przetwarza HTML do PDF? Czy też się mylę i nota ma swoją osobną wersję PDF jak faktura?

Reply:
> Właśnie co do PDF. Jak to wygląda? Bo faktury mają osobny moduł, który rysuje PDF, 

Noty obciążeniowe również:
https://github.com/chilek/lms-plus/blob/master/lib/LMSDocuments/LMSTcpdfDebitNote.php

Reply:
OK, to postaram się dorobić od razu commita do PDF.

Reply:
Siądę po pracy do tego i napiszę co i jak.

Reply:
Zmieniłem testowo **notes.type**=pdf i przy wyświetleniu lub utworzeniu nowej noty pojawia się HTTP ERROR 500, a w logu apache:
`PHP Fatal error:  Uncaught Error: Class 'LMSNote.htmlDebitNote' not found in /var/www/html/lms-dev/modules/note.php:132`

Coś jeszcze powinienem ustawić?

Reply:
Pewnie notes.template_file=standard

Reply:
OK. Dziękuję.